app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppState.kt

```diff
@@ -156,9 +163,24 @@ class NiaAppState(
             }
 
             when (topLevelDestination) {
-                FOR_YOU -> navController.navigateToForYou(topLevelNavOptions)
-                BOOKMARKS -> navController.navigateToBookmarks(topLevelNavOptions)
-                INTERESTS -> navController.navigateToInterests(null, topLevelNavOptions)
+                FOR_YOU -> niaNavigator.getNavigator(ForYouBaseNavigator::class.java)
```

Comment:
We now can't directly call `navigateToForYou(...)` - it is (assume) `internal` inside the `:impl` module.

Instead, we get the requisite navigator instance and call `navigateToRoute` on it.